### ISSUE TITLE
Change shutdown verbiage for config reload feature

### ISSUE THREAD

**Issue by suyograo** [elastic/logstash#4728]
With `--auto-reload` enabled, LS can dynamically apply config changes made to the file. The message emitted when a new config applies is misleading. Today it says:
```
Logstash shutdown completed
```

should be changed to 

```
Config change detected, pipeline <ID> has been shutdown
```

On startup of the pipeline, it should be:
```
New pipleine <ID> started with latest config
```

Also, I don't see any point in logging the new updated config . For users with large configs, this is going to be a lot of noise. Lets move this message to debug:

```
fetched new config for pipeline. upgrading .. {:pipeline => main .... 
```

**Comment by andrewvc** [comment]
+1 on this

**Comment by jsvd** [comment]
may I suggest changing `Logstash shutdown completed` to just `Pipeline <ID> has been shutdown`?
When quitting logstash we can't output "Config change detected..", doesn't make sense.

Also, when starting a pipeline, I suggest just `Pipeline <ID> started`.

As for logging the new pipeline config, changes should not happen very frequently and after 3, 4, 5, 10 reloads, if the config reloading is controlled from SIGHUP, the user will have no information about what pipeline config logstash is running at that moment.

**Comment by suyograo** [comment]
I am good with `Pipeline <ID> has been shutdown` and `Pipeline <ID> started` messages.

> As for logging the new pipeline config, changes should not happen very frequently and after 3, 4, 5, 10 reloads, if the config reloading is controlled from SIGHUP, the user will have no information about what pipeline config logstash is running at that moment.

This is ok for the first iteration, we might see some feedback from users and later change it.

**Comment by jsvd** [comment]
ok I'll update, and merge later today.

**Comment by jsvd** [comment]
closed by #4764
